packages/engine/lib/orchestrator/src/process/local.rs

```diff
@@ -89,7 +89,7 @@ impl process::Command for LocalCommand {
 
         let mut cmd = std::process::Command::new(process_path);
         cmd.arg("--experiment-id")
-            .arg(&self.experiment_id)
+            .arg(self.experiment_id.to_string())
```

Comment:
We we clone the id here? Can't we just pass the reference `&self.experiment.id` as before?

Reply:
ExperimentId is a `Uuid`, it needs to be converted to a string so that it can be passed